.github/workflows/cintegration-s3-apps-caller.yml

```diff
@@ -15,7 +15,7 @@ on:
 jobs:
   graasp-deploy-s3-apps-workflow:
     name: Graasp app task management
-    uses: graasp/graasp-deploy/.github/workflows/cintegration-s3-apps.yml@7b5dd84304458c3c5ab03cbbde059670f77b1687
+    uses: graasp/graasp-deploy/.github/workflows/cintegration-s3-apps.yml@v1
```

Comment:
That is a good idea to point to a tag, did not think about it. Maybe it would be worth changing this in other workflows too now that you are at it?

Reply:
Yes this is the approach we are using currently, the `v1` tag follows the latest major version 1. It is slightly less stable and secure than using the SHA but comes with less maintainer burden when needing to update (because of fixes)